docs/releases/4.0.md

```diff
@@ -47,3 +47,21 @@ The `PageRevision` model has been replaced with a generic `Revision` model. If y
  * Creation of `PageRevision` objects should be updated to create `Revision` objects using the page's `id` as the `object_id`, the default `Page` model's content type as the `base_content_type`, and the page's specific content type as the `content_type`.
  * Queries that use the `PageRevision.objects` manager should be updated to use the `Revision.page_revisions` manager.
  * Queries for `Page` objects that use `PageRevision.page_id` should be updated to cast the `Revision.object_id` to an integer before using it in the query.
+ * Access to `PageRevision.page` should be updated to `Revision.content_object`.
+
+If you are maintaining a package across multiple Wagtail versions that includes a model with a `ForeignKey` to the `PageRevision` model, you can create a helper function to correctly resolve the model depending on the installed Wagtail version, for example:
+
+```python
+from django.db import models
+from wagtail import VERSION as WAGTAIL_VERSION
+
+def get_revision_model():
```

Comment:
nitpick: always lint your code examples. this and the class definition need another blank line